<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapPackagesWidget</class>
 <widget class="QWidget" name="MapPackagesWidget">
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="installed">
      <attribute name="title">
       <string>Installed</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QListWidget" name="installedList"/>
       </item>
       <item>
        <widget class="WorldMapChooser" name="worldMap" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="changeDirectory">
           <property name="text">
            <string>Data Directory</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="load">
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteMap">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="switchSelection">
           <property name="text">
            <string>World Map</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="updatable">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <attribute name="title">
       <string>Update</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QListWidget" name="updateList">
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="check">
           <property name="text">
            <string>Check</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="update">
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="downloadable">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <attribute name="title">
       <string>Download</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,1,0">
       <item>
        <widget class="QToolButton" name="back">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="arrowType">
          <enum>Qt::LeftArrow</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="downloadBrowse"/>
       </item>
       <item>
        <widget class="QListWidget" name="downloadList">
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QComboBox" name="server"/>
           </item>
           <item>
            <widget class="QToolButton" name="addServer">
             <property name="text">
              <string>+</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="download">
           <property name="text">
            <string>Download</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WorldMapChooser</class>
   <extends>QWidget</extends>
   <header>worldmapchooser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>switchSelection</sender>
   <signal>toggled(bool)</signal>
   <receiver>installedList</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>774</x>
     <y>569</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>523</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>switchSelection</sender>
   <signal>toggled(bool)</signal>
   <receiver>worldMap</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>750</x>
     <y>568</y>
    </hint>
    <hint type="destinationlabel">
     <x>693</x>
     <y>443</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
